.travis.yml

```diff
@@ -0,0 +1,12 @@
+sudo: false
+language: node_js
+node_js:
+  - 0.10
+cache:
+  directories:
+  - node_modules
+before_script:
+  - npm install -g grunt-cli
+  - npm install -g mocha
+after_script:
+  - istanbul cover _mocha --report lcovonly && cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js
```

Comment:
I haven't noticed it on any other repos yet, maybe I've been focussed on the closed repos. We are now tracking our coverage on graphite with for example https://github.com/holidayextras/revolver/blob/staging/circle.yml#L23